conference.js

```diff
@@ -1045,6 +1062,10 @@ export default {
                 (typeof config.hosts.jirecon != "undefined"))?
                 "jirecon" : "colibri";
         }
+        // Initialize in audio only mode
```

Comment:
there is some problem there I don't remember what it was right now, but I'll dig into that deeper